### PR TITLE
Humble Fix Bundle

### PR DESCRIPTION
**Pull request by Core0verload**
This PR fixes bugs.
- Fixes any blood pumped out of humans always rejecting.
- Fixes "A+" blood not accepting "A-".
- Fixes AutoDrobe disassembly dropping Getmore Chocolate Corp board.
- Fixes plasma cutters being useless for mining because of changes in projectile ranging.
- Fixes plasma cutters being useless for lavaland mob hunt because lavaland has air.
- Fixes lava burning projectiles.
- Removes one unused file.
- Changes all `/obj/item/effect` to `/obj/effect` for the sake of consistency.
- Changes material datums to be less insane.

Yes, I checked material datums, they had no reason to set all those vars in new instead of declaration.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Good stuff, thanks.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -998,7 +998,7 @@
 	visible_message("<B><span class='danger'>The tendril writhes in pain and anger and the earth around it begins to split! Get back!</span></B>")
 	visible_message("<span class='danger'>A chest falls clear of the tendril!</span>")
 	playsound(get_turf(src),'sound/effects/tendril_destroyed.ogg', 200, 0, 50, 1, 1)
-	spawn(50)
+	spawn(75)
```
Comment (lordpidey) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: What's this change for?

Comment (Core0verload) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: For unchecked file on commit.

**Comment by ike709** [comment]
>Fixes AutoDrobe disassembly dropping Getmore Chocolate Corp board.

I'm pretty sure this has been a bug longer than I've been playing SS13 (3 years), so good job for finally fixing it.

**Comment by optimumtact** [comment]
You've changed a bunch of damage strengths, can you expand on why?, I'm loathe to merge otherwise.

**Comment by evsey9** [comment]
I will pay 1$ for this bundle.
Gotta get those steam keys, you know.

File: code/modules/projectiles/projectile/special.dm [hunk 2]
```diff
@@ -227,22 +227,24 @@
...
 /obj/item/projectile/plasma/on_hit(atom/target)
 	. = ..()
 	if(istype(target, /turf/closed/mineral))
 		var/turf/closed/mineral/M = target
 		M.gets_drilled(firer)
-		range = max(range - 1, 1)
-		return -1
+		Range()
+		if(range > 0)
+			return -1
```
Comment (phil235) [inline code/modules/projectiles/projectile/special.dm]: can you explain this change? Did you test it locally?

Comment (Core0verload) [inline code/modules/projectiles/projectile/special.dm]: Yes, I did. `Range()` is just a proc for doing projectile range checks.

Comment (phil235) [inline code/modules/projectiles/projectile/special.dm]: I know, but why did you change the code to that? Afaik Range() is already called a bit after on_hit() in projectile code. Is your change fixing a specific bug?

**Comment by optimumtact** [comment]
I'm going to trust you didn't make the plasma cutters too OP.

Woe on me if we have to nerf this.

**Comment by phil235** [comment]
@optimumtact My concerns about the Range() call were not addressed...

File: code/modules/projectiles/projectile/special.dm (hunk 2 above)
Comment (Core0verload) [inline code/modules/projectiles/projectile/special.dm]: Range() is called when a proj travels a tile, calling it twice is necessary to make proj less powerful while punching through solid mineral walls than when flying through air. It didn't used range before, but someone has changed it to range. And this resulted in fuckery and plasma cutter being so weak it bordered on useless.

**Comment by Core0verload** [comment]
If there would be "OP NERF PLZ" cries, ping me and I'll nerf it myself. The changes are to bring cutter in line with other RnD-only mining tools. Yed, super-turbo-hyper-mega KA, I look at you.
